R/pkg/R/generics.R

```diff
@@ -1046,8 +1084,8 @@ setGeneric("ntile", function(x) { standardGeneric("ntile") })
 #' @export
 setGeneric("n_distinct", function(x, ...) { standardGeneric("n_distinct") })
 
-#' @rdname percent_rank
-#' @export
+# @rdname percent_rank
```

Comment:
try changing the S4 signature from `(x = missing)` to `(missing)` - i think that should eliminate the need for a dummy parameter.